tests/data/placebo/test_ebs_snapshot_copy_volume_tags/ec2.CreateSnapshot_1.json

```diff
@@ -0,0 +1,41 @@
+{
+  "status_code": 200,
+  "data": {
+    "Description": "",
+    "ResponseMetadata": {
```

Comment:
This key should typically be an empty dict as we don't use it in the test replays. 
`"ResponseMetadata": {}`
Curious, did you record this? It should have been stripped in the recording.

Reply:
Fixed, thanks.